src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import ChatEntry from './ChatEntry.js';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, likeMessage}) => {
+    const chatEntries= (entries) => {
+        return entries.map((message) => {
+            return(
+                <ChatEntry
+                    key={message.id}
+                    id={message.id}
+                    sender={message.sender}
+                    body={message.body}
+                    timeStamp={message.timeStamp}
+                    liked={message.liked}
+                    likeMessage={likeMessage}
+                />
+
+            );
+        });
+    };
```

Comment:
```suggestion
    const chatEntries = entries.map((message) => {
        return(
            <ChatEntry
                key={message.id}
                id={message.id}
                sender={message.sender}
                body={message.body}
                timeStamp={message.timeStamp}
                liked={message.liked}
                likeMessage={likeMessage}
            />
        );
    });
```
No need to wrap the map logic in a function. You can refactor this to just run the `entries.map(...)` logic and store the result in a variable.